src/components/delete-project/delete-project-modal.html

```diff
@@ -14,7 +14,7 @@ <h1 class="modal-title">Are you sure you want to delete the project '<strong>{{p
         undone</strong>.  Make sure this is something you really want to do!
       </p>
       <div ng-show="typeNameToConfirm">
-        <p>Type the name of the project to confirm.</p>
+        <p>Type <strong>{{project | displayName}}</strong> to confirm deletion.</p>
```

Comment:
I might suggest

```html
Type <strong>{{project.metadata.name}}</strong> to confirm deletion.
```

Either name or display name will work, but name is probably easier to type. Also two projects can have the same display name, whereas name is unambiguous.

Reply:
Changed.